src/xcode/ENA/ENA/Source/Models/Profile/RapidTestProfile.swift

```diff
@@ -0,0 +1,85 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+
+struct RapidTestProfile: Codable {
+
+	// MARK: - Init
+
+	init(
+		forename: String? = nil,
```

Comment:
```suggestion
		firstName: String? = nil,
```
More commonly used and also consistent with TestedPerson on AntigenTest